### ISSUE TITLE
ofToBool function is not working on windows

### ISSUE THREAD

**Issue by 60-hz** [cuinjune/Ofelia#42]
ofToBool return always false on windows systems.

Tested with ofelia V4.0.0 and windows10 64bit.

**Comment by cuinjune** [comment]
As I tested on macOS, `ofToBool()` seems to work fine.

```
tgl
|
ofelia f;
print(ofToBool(a));
```

And you can basically do the same with the following if you are simply trying to convert float to boolean:

```
tgl
|
ofelia f;
print(a ~= 0);
```

**Comment by emviveros** [comment]
I can confirm the misbehavior of ofToBool() in windows

**Comment by grabanton** [comment]
Yes. It always return false. Same setup: ofelia v4.0.0 and windows 10 64bit
